awscli/examples/networkflowmonitor/tag-resource.rst

```diff
@@ -0,0 +1,9 @@
+**To add a tag to the specified resource**
+
+The following ``tag-resource`` example adds a tag to the monitor in the specified account. ::
+
+    aws networkflowmonitor tag-resource \
+        --resource-arn arn:aws:networkflowmonitor:us-east-1:123456789012:monitor/Demo \
+        --tags Key=stack,Value=Production  
+
+This command produces no output.
```

Comment:
Every example needs a for more information link.
https://aws.github.io/aws-cli/docs_styleguide.html#for-more-information-link